From a3b6c90f79e706ef83470e70c95a99f0d84726b2 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <aisheng.dong@freescale.com>
Date: Wed, 16 Dec 2015 11:23:28 +0800
Subject: [PATCH 0779/5242] MLK-12016-4 dts: imx6: remove flexcan for pin
 conflict devices

commit  d6f048094ed8e04eddcd685d307d807170be8ac7 from
https://source.codeaurora.org/external/imx/linux-imx.git

Removing flexcan for pin conflict devices such as ecspi/gpmi.

Signed-off-by: Dong Aisheng <aisheng.dong@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6dl-sabreauto-ecspi.dts     |    5 +----
 arch/arm/boot/dts/imx6dl-sabreauto-gpmi-weim.dts |    5 +----
 arch/arm/boot/dts/imx6q-sabreauto-ecspi.dts      |    5 +----
 arch/arm/boot/dts/imx6q-sabreauto-gpmi-weim.dts  |    5 +----
 arch/arm/boot/dts/imx6qp-sabreauto-ecspi.dts     |    5 +----
 arch/arm/boot/dts/imx6qp-sabreauto-gpmi-weim.dts |    5 +----
 6 files changed, 6 insertions(+), 24 deletions(-)

diff --git a/arch/arm/boot/dts/imx6dl-sabreauto-ecspi.dts b/arch/arm/boot/dts/imx6dl-sabreauto-ecspi.dts
index 23d362c..45ae162 100644
--- a/arch/arm/boot/dts/imx6dl-sabreauto-ecspi.dts
+++ b/arch/arm/boot/dts/imx6dl-sabreauto-ecspi.dts
@@ -13,13 +13,10 @@
 	status = "okay";
 };
 
-/* remove the #if 0 after flexcan is ready */
-#if 0
-&flexcan2 {
+&can2 {
 	/* max7310_c on i2c3 is gone */
 	status = "disabled";
 };
-#endif
 
 &i2c3 {
 	/* pin conflict with ecspi1 */
diff --git a/arch/arm/boot/dts/imx6dl-sabreauto-gpmi-weim.dts b/arch/arm/boot/dts/imx6dl-sabreauto-gpmi-weim.dts
index bceeb39..ad2e937 100644
--- a/arch/arm/boot/dts/imx6dl-sabreauto-gpmi-weim.dts
+++ b/arch/arm/boot/dts/imx6dl-sabreauto-gpmi-weim.dts
@@ -13,13 +13,10 @@
 	status = "disabled";
 };
 
-/* remove #if 0 after flexcan is ready */
-#if 0
-&flexcan2 {
+&can2 {
 	/* max7310_c on i2c3 is gone */
 	status = "disabled";
 };
-#endif
 
 &gpmi {
 	status = "okay";
diff --git a/arch/arm/boot/dts/imx6q-sabreauto-ecspi.dts b/arch/arm/boot/dts/imx6q-sabreauto-ecspi.dts
index 4fbb207..3cf99ed 100644
--- a/arch/arm/boot/dts/imx6q-sabreauto-ecspi.dts
+++ b/arch/arm/boot/dts/imx6q-sabreauto-ecspi.dts
@@ -13,13 +13,10 @@
 	status = "okay";
 };
 
-/* remove #if 0 after flexcan is ready */
-#if 0
-&flexcan2 {
+&can2 {
 	/* max7310_c on i2c3 is gone */
 	status = "disabled";
 };
-#endif
 
 &i2c3 {
 	/* pin conflict with ecspi1 */
diff --git a/arch/arm/boot/dts/imx6q-sabreauto-gpmi-weim.dts b/arch/arm/boot/dts/imx6q-sabreauto-gpmi-weim.dts
index 269d362..579aeb2 100644
--- a/arch/arm/boot/dts/imx6q-sabreauto-gpmi-weim.dts
+++ b/arch/arm/boot/dts/imx6q-sabreauto-gpmi-weim.dts
@@ -13,13 +13,10 @@
 	status = "disabled";
 };
 
-/* remove #if 0 after flexcan is ready */
-#if 0
-&flexcan2 {
+&can2 {
 	/* max7310_c on i2c3 is gone */
 	status = "disabled";
 };
-#endif
 
 &gpmi {
 	status = "okay";
diff --git a/arch/arm/boot/dts/imx6qp-sabreauto-ecspi.dts b/arch/arm/boot/dts/imx6qp-sabreauto-ecspi.dts
index ace6da9..8846739 100644
--- a/arch/arm/boot/dts/imx6qp-sabreauto-ecspi.dts
+++ b/arch/arm/boot/dts/imx6qp-sabreauto-ecspi.dts
@@ -13,13 +13,10 @@
 	status = "okay";
 };
 
-/* remove #if 0 after flexcan is ready */
-#if 0
-&flexcan2 {
+&can2 {
 	/* max7310_c on i2c3 is gone */
 	status = "disabled";
 };
-#endif
 
 &i2c3 {
 	/* pin conflict with ecspi1 */
diff --git a/arch/arm/boot/dts/imx6qp-sabreauto-gpmi-weim.dts b/arch/arm/boot/dts/imx6qp-sabreauto-gpmi-weim.dts
index ce910b6..b91ebad 100644
--- a/arch/arm/boot/dts/imx6qp-sabreauto-gpmi-weim.dts
+++ b/arch/arm/boot/dts/imx6qp-sabreauto-gpmi-weim.dts
@@ -13,13 +13,10 @@
 	status = "disabled";
 };
 
-/* remove #if 0 after flexcan is ready */
-#if 0
-&flexcan2 {
+&can2 {
 	/* max7310_c on i2c3 is gone */
 	status = "disabled";
 };
-#endif
 
 &gpmi {
 	compatible = "fsl,imx6qp-gpmi-nand";
-- 
1.7.9.5

